client/src/app/__tests__/TabsProviderSpec.js

```diff
@@ -424,17 +424,69 @@ describe('TabsProvider', function() {
     expect(providers['empty']).to.exist;
   });
 
+  describe('#getProviderNames', function() {
 
-  it('#getProviderNames', function() {
+    it('should return default provider names', function() {
 
-    // given
-    const tabsProvider = new TabsProvider();
+      // given
+      Flags.init({});
+      const tabsProvider = new TabsProvider();
 
-    // when
-    const providerNames = tabsProvider.getProviderNames();
+      // when
+      const providerNames = tabsProvider.getProviderNames();
 
-    // then
-    expect(providerNames).to.eql([ 'BPMN', 'CMMN', 'DMN', 'FORM' ]);
+      // then
+      expect(providerNames).to.eql(['BPMN', 'DMN', 'FORM']);
```

Comment:
Let's try to add spaces inside the array entries. I remember we recently added a [linting rule for that](https://github.com/bpmn-io/eslint-plugin-bpmn-io/commit/8dc9959e658b96b013780a3105713b5f3e5ce76b), but seems like it did not end yet inside the Modeler.

```js
expect(providerNames).to.eql([ 'BPMN', 'DMN', 'FORM' ]);
```

Same goes for the other name arrays below.